cmd/otelcontribcol/go.mod

```diff
@@ -2,7 +2,9 @@
 
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontribcol
 
-go 1.20
+go 1.21.1
```

Comment:
what's the reason for these changes?

Reply:
Hmmm, I ran a go mod tidy and go get. 
Not sure if that updates these?

Reply:
Updated the paths